resources/admin/classes/domain/Resource.php

```diff
@@ -1366,7 +1366,7 @@ public function getAlphabeticalList() {
 		$alphArray = array();
 		$result = $this->db->query("SELECT DISTINCT UPPER(SUBSTR(TRIM(LEADING 'The ' FROM titleText),1,1)) letter, COUNT(SUBSTR(TRIM(LEADING 'The ' FROM titleText),1,1)) letter_count
 								FROM Resource R
-								GROUP BY SUBSTR(TRIM(LEADING 'The ' FROM titleText),1,1)
+								GROUP BY SUBSTR(TRIM(LEADING 'The ' FROM titleText),1,1), titleText
```

Comment:
Looks like a decision needs to be made here between two different solutions. @remocrevo or @veggiematts do either of you have the time to modify this to use the solution that prevents the warning? If not would it be acceptable to accept this as is and then add an issue for making this modification later?